CONTRIBUTING.md

```diff
@@ -79,17 +84,33 @@ You can access the app [localhost:3000](http://localhost:3000). Whatever you cha
   * If there is already such a discussion on the issue, you can go ahead and pick this up
   * If not, please first leave a comment on how you want to implement it and wait for contributors' feedback
 
+### Commits
+* Commit title should be short and descriptive
+  * title (or summary line) is the first line of the commit message
+  * that says what the commit is doing
+  * in no more than 50 characters
+  * starting with a word like 'Fix' or 'Add' or 'Change'
+  * **without** a period (.) at the end
+  * followed by a blank line
+* Commit messages are
+  * up to 72 characters
+  * with break lines
+* Reference the issue(s) the commit closes
+  * https://help.github.com/articles/closing-issues-via-commit-messages
+  * If you haven't done so since the beginning, you should reference the issue when you squash your commits
+
 ### Pull Requests workflow
-Please open a PR only when you have finished coding, and your changes are ready to be reviewed for merging.
+Please open a pull request (PR) only when you have finished coding, and your changes are ready to be reviewed for merging.
 
 * Title
   * Include a comprehensive title about what this PR is doing
   * Referencing the issue number on the PR title is not giving any information about what this PR is about
-  * The title should be short (50 characters maximum). You can add more information in the description
+  * The title should be short (50 characters maximum); you can add more information in the description
```

Comment:
To be honest I was thinking about the commit and not the PR when writing this, as you already added in the commit section it can be removed.